package.json

```diff
@@ -111,17 +112,17 @@
     "opn": "^4.0.1",
     "opn-cli": "^3.1.0",
     "postcss-prefix-selector": "^1.6.0",
-    "react": "^15.3.1",
+    "raf": "^3.3.2",
```

Comment:
What was the reason for bringing in `raf` in this PR?

Reply:
Covered this here: https://github.com/seek-oss/seek-style-guide/pull/333#discussion_r141238096

Reply:
Just saw it's usage further down.... as you were